src/main/resources/db/migration/V2_create_users_table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE User(
+     id SERIAL NOT NULL DEFAULT PRIMARY KEY,
+     username VARCHAR(20) NOT NULL,
+     password VARCHAR(30) NOT NULL,
+     firstname VARCHAR(20) NOT NULL,
```

Comment:
Column name are all in snake_case (refer to https://www.sqlshack.com/learn-sql-naming-conventions/)